### PR TITLE
Finished companion partial class and companion models

### PR DESCRIPTION
**Pull request by dgilliam2022**
Check it y'all, I added companion partial class and companion models class

### CODE REVIEW

File: Cosminis/Models/Companion.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                {MoodCompanion.Excited, "Excited"},
+                {MoodCompanion.Chill, "Chill"}
+
+            };
+
+            if(dictMood.ContainsKey(Mood))
+            {
+                return dictMood[Mood];
+            }
+            return "NotFound";
```
Comment (SenLongYu) [inline Cosminis/Models/Companion.cs]: Can you make this lint throw an exception?

**Review by SenLongYu — CHANGES_REQUESTED** [review]

**Comment by TheFakeLorLyons** [comment]
LGTM

**Review by SenLongYu — APPROVED** [review]
LGTM

File: Cosminis/Models/Companion.cs (hunk 1 above)
Comment (TheFakeLorLyons) [reply Cosminis/Models/Companion.cs]: ^

**Review by TheFakeLorLyons — APPROVED** [review]
LGTM

**Pull request merged**